state/state.go

```diff
@@ -108,6 +108,8 @@ type ReleaseSpec struct {
 	// The 'env' section is not really necessary any longer, as 'set' would now provide the same functionality
 	EnvValues []SetValue `yaml:"env"`
 
+	ValuePath string `yaml:"valuePath"`
```

Comment:
Very nit but I would personally prefer `ValuesPathPrefix `yaml:"valuesPathPrefix"`, so that it is clearer that each entry in `values:` is appended to this.

Reply:
Thank @mumoshu, that is a good name, I updated code, run `ftm` and try to run `make pristine` from my local but the result is different with the CI
```
make pristine
go generate github.com/roboll/helmfile github.com/roboll/helmfile/args github.com/roboll/helmfile/environment github.com/roboll/helmfile/event github.com/roboll/helmfile/helmexec github.com/roboll/helmfile/state github.com/roboll/helmfile/tmpl github.com/roboll/helmfile/valuesfile
go fmt github.com/roboll/helmfile github.com/roboll/helmfile/args github.com/roboll/helmfile/environment github.com/roboll/helmfile/event github.com/roboll/helmfile/helmexec github.com/roboll/helmfile/state github.com/roboll/helmfile/tmpl github.com/roboll/helmfile/valuesfile
git ls-files --exclude-standard --modified --deleted --others | diff /dev/null -
➜  helmfile git:(feature/add-common-values-path) echo $?
0
```
Can you check from CI setup env?

Reply:
I guess that's due to a golang version difference. I'll take a look! Thanks.

Reply:
It worked 🎉

Reply:
Thank so much for the update, my fault for the missing check version in circleCI :)